server/data/data-library/src/test/java/org/apache/james/rrt/lib/AbstractRecipientRewriteTableTest.java

```diff
@@ -328,6 +328,25 @@ public void sortMappingsShouldPutDomainAliasFirstWhenVariousMappings() {
                         .build());
     }
 
+    @Test
+    public void addMappingShouldThrowWhenMappingAlreadyExists() throws Exception {
```

Comment:
shouldThrow? but it has assertThat?

Reply:
yes, some tests methods are swallowing exceptions and transform them to boolean (false in case of exception or failure). I made the changes to have proper test changes, but they fail with hbase because of the incompatibility of Guave version (cannot found MoreObject). I don't want to go deeper into this, so I let it like that.

Reply:
finally the fix was needed, so fixed